src/src/Helpers/StringConverter_Numerical.cpp

```diff
@@ -116,6 +116,17 @@ String trimTrailingZeros(const String& value) {
 
 }
 
+/**
+ * Helper: Convert an integer to string, but return an empty string for 0, to save a little space in settings
+ */
+String toStringNoZero(int64_t value) {
+  if (value != 0) {
+    return toString(value, 0);
```

Comment:
`toString` is using a float.
So you're loosing resolution.
Better use `ll2String`

Reply:
Fixed